src/Sarif/CodeGenHints.json

```diff
@@ -281,6 +387,24 @@
           "SuppressedExternally"
         ]
       }
+    },
+    {
+      "kind": "AttributeHint",
+      "arguments": {
+        "namespaceName": "Newtonsoft.Json",
+        "typeName": "JsonConverter",
+        "arguments": [ "typeof(FlagsEnumConverter)" ]
+      }
+    }
+  ],
+  "Result.WorkItemUris": [
+    {
+      "kind": "AttributeHint",
+      "arguments": {
+        "namespaceName": "Newtonsoft.Json",
+        "typeName": "JsonConverter",
```

Comment:
>JsonConverter [](start = 21, length = 13)

Does the converter work on the array elements when you apply the attribute to an array? Quick search didn't turn up the answer. Is this tested?

Reply:
Yes, this works. The UriConverter actually operates against List<uri> as well.

---
In reply to: [221316128](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221316128) [](ancestors = 221316128)